docs/JSON-RPC-API/Using-JSON-RPC-API.md

```diff
@@ -64,25 +85,19 @@ Send individual requests as a JSON data package at each prompt:
 > {"jsonrpc":"2.0","method":"web3_clientVersion","params":[],"id":53}
 ```
 
-<<<<<<< HEAD:docs/JSON-RPC-API/Using-JSON-RPC-API.md
-The [RPC Pub/Sub methods](../Using-Pantheon/RPC-PubSub.md) can also be used over WebSockets.
-
 !!! note 
     `wscat` does not support headers. [Authentication](Authentication.md) requires an authentication token to be passed in the 
     request header. To use authentication with WebSockets, an app that supports headers is required. 
 
-## API Methods Enabled by Default
-=======
 ### API Methods Enabled by Default
->>>>>>> 149c0c24631231f8a96f5740534d309774e99ff5:docs/Reference/Using-JSON-RPC-API.md
 
 The `ETH`, `NET`, and `WEB3` API methods are enabled by default. 
 
 Use the [`--rpc-http-api`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-api) or [`--rpc-ws-api`](../Reference/Pantheon-CLI-Syntax.md#rpc-ws-api) 
-options to enable the `ADMIN` ,`CLIQUE`,`DEBUG`, `IBFT` and `MINER` API methods.
+options to enable the `ADMIN` ,`CLIQUE`,`DEBUG`, `EEA`, `IBFT`, `MINER`, and `PERM` API methods.
 
-!!! note
-    IBFT 2.0 is under development and will be available in v1.0. 
+!!!note
+    EEA methods are for privacy features. Privacy features are under development and will be available in v1.1.  
```

Comment:
Is it clear enough that `EEA` and `PERM` = Privacy features/methods?

Reply:
I'm going to leave this as is for now because it's not available until v1.1. But will keep in mind the need to make this clearer when doing the privacy doc.